tests/test_flatten.py

```diff
@@ -0,0 +1,36 @@
+from unittest.mock import patch
+
+from geopandas import GeoDataFrame
+from pandas import DataFrame
+from pytest import raises
+from shapely.geometry import Point
+
+from crossfire import flatten
+
+DICT_DATA = [
+    {"answer": 42, "contextInfo": {"context1": "info1", "context2": "info2"}}
+]
+PD_DATA = DataFrame(DICT_DATA)
+geometry = [Point(4, 2)]
+GEOPD_DATA = GeoDataFrame(DICT_DATA, crs="EPSG:4326", geometry=geometry)
+
+
+# test the existence of the flatten function
+assert flatten
```

Comment:
We don't need that.  If there's no `flatten` at this point, line 8 would have already failed at import. Also, we would have a name error when we call the function in lines 24 and 30.

```suggestion
```